book/Zapp-Pipes/12.-Datasource-for-rokuTV.md

```diff
@@ -0,0 +1,83 @@
+# Data Source Provided (DSP) for Roku-TV
+
+**This documentation is in beta**
+
+The Roku-TV system cannot use the current DSP solution, because the javascript code can't be parsed by Roku. This documentation explain how to solve this problem in a few steps.
+
+Applicaster provide a plugin for Roku-tv, this plugin parse the DSP `manifest.json`. This Roku plugin read the endpoint's described inside the `manifest`. Then the Roku App can call an external (rest) service, in order to receive the feed data from the client API.
+
+This is the steps to follow for enable the data source provided inside roku tv :
```

Comment:
These are the steps to follow so as to enable the data source provider inside Roku TV: